reference-architectures/watsonx-saas-da.md

```diff
@@ -0,0 +1,146 @@
+{: caption="Figure 1. Watsonx AI SaaS Automation with Assistant and Governance deployable architecture" caption-side="bottom"}{: external download="watsonx-saas-da.svg"}
+
+The Watsonx AI SaaS deployable architecture lays down the services shown in the WatsonX services box and an instance of {{site.data.keyword.cos_full_notm}} in a target {{site.data.keyword.Bluemix_notm}} account, resource group, and region. Then, it automatically configures a starter project that grants access to an existing {{site.data.keyword.Bluemix_notm}} user, for example, an AI researcher. As a result, that user can
+log into the {{site.data.keyword.IBM_notm}} watsonx starter project, and start working.
+
+You can add or deploy one or more of the services from the optional services section at a later date during the lifetime of the deployable architecture.
+
+## Design concepts
+{: #design-concepts}
+
+![Design requirements for Watsonx AI SaaS deployable architecture](heat-map.svg "Design requirements"){: caption="Figure 2. Scope of the design requirements" caption-side="bottom"}
+
+## Requirements
+{: #requirements}
+
+The following table outlines the requirements that are addressed in this architecture.
+
+| Aspect | Requirements |
+| -------------- | -------------- |
+| Enterprise applications | Lay down and grant access to the {{site.data.keyword.IBM_notm}} Watsonx Artificial Intelligence and Governance platform. |
+| Storage            | Provide storage that meets the application performance and security requirements |
+| Security           | * Protect boundaries against denial of service and application layer attacks. \n * Encrypt all application data in transit and at rest to protect from unauthorized disclosure. \n * Encrypt all security data (operational and audit logs) to protect from unauthorized disclosure. \n * Protect secrets through their entire lifecycle and secure them using access control measures. |
+| Resiliency         | * Support application availability targets and business continuity policies. \n * Ensure availability of the services in the event of planned and unplanned outages \n * Provide highly available storage artificial intelligence assets. |
+| Service Management | Monitor audit logs to track changes and detect potential security problems. |
+{: caption="Table 1. Requirements" caption-side="bottom"}
+
+## Components
+{: #components}
+
+The following table outlines the services used in the architecture for each aspect.
+
+| Aspects | Architecture components | How the component is used |
+| -------------- | -------------- | -------------- |
+| Storage | Cloud Object Storage | Stores artificial intelligence {{site.data.keyword.IBM_notm}} Watsonx data assets managed by Watson Machine Learning and Watson Studio services. |
```

Comment:
Should Watsonx be upper or lowercase? Seeing different variations.

Reply:
When referred to the watsonx should be always `IBM watsonx`.  
When referred to classic watson services should be Watson. I'll fix.